README.md

```diff
@@ -162,18 +202,18 @@ functions in other crates.
 The implementation is memory-safe, and contains no significant
 `unsafe` code.  The SIMD backend uses `unsafe` internally to call SIMD
 intrinsics.  These are marked `unsafe` only because invoking them on an
-inappropriate CPU would cause `SIGILL`, but the entire backend is only
+inappropriate CPU would cause `SIGKILL`, but the entire backend is only
```

Comment:
```suggestion
inappropriate CPU would cause `SIGILL`, but the entire backend is only
```

.. yeah *NIX processes can become ill :roboWithThermometer: when we throw illegalities at them :woman_shrugging: